File: hello.rs
// Jacobus Burger (2022)
// Rust
// https://www.rust-lang.org/
use std::io;

fn main() -> io::Result<()> {
    let mut name = String::new();
    io::stdin().read_line(&mut name);
    println!("Hello, {}!", name);
    Ok(())
}

File: hello.f90
! Jacobus Burger (2023)
! Fortran 90
! https://fortran-lang.org/
program hello
  character (len = 256) :: name
  read (*,*) name  ! note: will only read first word on line
  write (*,*) 'Hello, ', trim(name), '!'
end program hello
